examples/aws/eks/teleport.yaml

```diff
@@ -0,0 +1,44 @@
+export TELEPORT_PUBLIC_DNS_NAME="[teleport-proxy.example.com]
+export TELEPORT_CLUSTER_NAME="[teleport-cluster-name]"
+    cat > teleport.yaml << EOF
+# By default, this file should be stored in /etc/teleport.yaml
+teleport:
+  nodename: $TELEPORT_PUBLIC_DNS_NAME
+  cluster_name: $TELEPORT_CLUSTER_NAME
+  data_dir: /var/lib/teleport
+  log:
+    output: stderr
+    severity: ERROR
+  storage:
+    type: dir
+  auth_servers:
+  - 127.0.0.1:3025
+  log:
+    output: stderr
+    severity: INFO
+  auth_service:
+    enabled: "yes"
+    authentication:
+        type: github
+        # IP and the port to bind to. Other Teleport nodes will be connecting to
+        # this port (AKA "Auth API" or "Cluster API") to validate client
+        # certificates
+        listen_addr: 0.0.0.0:3025
```

Comment:
This is the default - are we just adding this so people are aware of the port?

Reply:
yeah, I agree. Since it's default I've removed it.